### PR TITLE
Implement test for ssl functionality

### PR DESCRIPTION
**Pull request by svicervlad**
Issue: #30

### CODE REVIEW

File: .github/workflows/ssl_test.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+    - name: Check drupal.devel domain work
+      run: curl http://drupal.devel | grep "Drupal"
+
+    - name: Generate certificates for next step
+      uses: kofemann/action-create-certificate@v0.0.4
+      with:
+        hostcert: 'cert.pem'
+        hostkey:  'private.key'
+        cachain:  'ca-chain.pem'
```
Comment (danylevskyi) [inline .github/workflows/ssl_test.yml:69]: It's redundant. Let's remove it.

File: .github/workflows/ssl_test.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    - name: Run main playbook with ssl configuration
+      uses: dawidd6/action-ansible-playbook@v2
+      with:
+        playbook: main.yml
+        directory: .drupal-dockerizer
+        options: |
+          --verbose
+
+    - name: Check drupal.devel domain work with ssl
```
Comment (danylevskyi) [inline .github/workflows/ssl_test.yml:88]:
```suggestion
    - name: Check if the site can be accessed via SSL
```

File: .github/workflows/ssl_test.yml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    - name: Run drush-commands playbook
+      uses: dawidd6/action-ansible-playbook@v2
+      with:
+        playbook: drush-commands.yml
+        directory: .drupal-dockerizer
+        options: |
+          --verbose
+
+    - name: Check drupal.devel domain work
```
Comment (danylevskyi) [inline .github/workflows/ssl_test.yml:61]:
```suggestion
    - name: Check if the site can be accessed without SSL
```

File: .github/workflows/ssl_test.yml [hunk 4]
```diff
@@ -0,0 +1,89 @@
+name: Check ssl functionality
```
Comment (danylevskyi) [inline .github/workflows/ssl_test.yml:1]:
```suggestion
name: Check SSL connection
```

**Review by danylevskyi — CHANGES_REQUESTED** [review]

File: .github/workflows/ssl_test.yml (hunk 1 above)
Comment (svicervlad) [reply .github/workflows/ssl_test.yml:69]: It`s required option for this action https://github.com/kofemann/action-create-certificate/blob/master/action.yml#L18

**Review by danylevskyi — APPROVED** [review]

**Pull request merged**